src/vmshepherd/http/rpc_api.py

```diff
@@ -113,7 +116,9 @@ def wrap(self, *args, **kwargs):
         # check in cache
         for vm in preset.vms:
             if vm_id == vm.id:
-                return vm.ip[0]
+                return {'vm_ip': vm.ip[0]}
         # retrieve real time data
         vm_info = await preset.iaas.get_vm(vm_id)
-        return vm_info.ip[0]
+        return {
+            'vm_ip': vm_info.ip[0]
+        }
```

Comment:
the other way around would be (IMVHO) better change - leave api as is here and match ours internal stuff to this state.